apps/remix-ide-e2e/src/tests/debugger.test.ts

```diff
@@ -14,8 +14,11 @@ module.exports = {
 
   'Should launch debugger #group1': function (browser: NightwatchBrowser) {
     browser.addFile('blah.sol', sources[0]['blah.sol'])
+      .pause(4000)
+      .clickLaunchIcon('solidity').click('*[data-id="compilerContainerCompileBtn"]')
```

Comment:
Please add comment explaining why you need to compile it manually when auto compile is on